app/iht/constants/Constants.scala

```diff
@@ -70,6 +71,58 @@ object Constants {
   val AppSectionEstateDebts = "estateDebts"
   val AppSectionEstateGifts = "estateGifts"
 
+  // Fragment Identifiers - used to return user to position in list
+  val AppSectionPropertiesID = "properties-buildings-and-land"
```

Comment:
I think these should be externalised to a file in the conf folder. They are visible to the user and this way, as with the contents of the routes file, they can easily be changed in the future without a recompilation of the application. Maybe create a new text file in conf called something like fragment-ids.